[DataFrame] update to 1.22.0

Library name: [hosseinmoein](https://github.com/hosseinmoein)/[DataFrame](https://github.com/hosseinmoein/DataFrame)

New version number: 1.22.0

Other information that may be useful (release notes, etc...)

in vcpkg:
>PS D:\soft\vcpkg> vcpkg search DataFrame
>dataframe                1.19.0           This is a C++ statistical library that provides an interface similar to Pa...

if I remove DataFrame and then run ```vcpkg install DataFrame --head```, I get a error:

> PS D:\soft\vcpkg> vcpkg remove dataframe
> The following packages are not installed:    dataframe:x86-windows
> Removing 1/1 dataframe:x86-windows
> PS D:\soft\vcpkg> vcpkg install DataFrame --head
> Computing installation plan...
> The following packages will be built and installed:
>     dataframe[core]:x86-windows -> 1.19.0 (+HEAD)
> Restored 0 package(s) from C:\Users\king\AppData\Local\vcpkg\archives in 197.2 us. Use --debug to see more details.
> Installing 1/1 dataframe:x86-windows...
> Building dataframe[core]:x86-windows from HEAD...
> -- Using HTTP(S)_PROXY in environment variables.
> -- Downloading https://api.github.com/repos/hosseinmoein/DataFrame/git/refs/heads/master -> hosseinmoein-DataFrame-master.version...
> -- Downloading https://github.com/hosseinmoein/DataFrame/archive/a851f3f4e9cfc1be1aa2c3407c8b119961acef64.tar.gz -> hosseinmoein-DataFrame-a851f3f4e9cfc1be1aa2c3407c8b119961acef64.tar.gz...
> -- Cleaning sources at D:/soft/vcpkg/buildtrees/dataframe/src/head/9961acef64-2ad6077aff.clean. Use --editable to skip cleaning for the packages you specify.
> -- Extracting source D:/soft/vcpkg/downloads/hosseinmoein-DataFrame-a851f3f4e9cfc1be1aa2c3407c8b119961acef64.tar.gz
> -- Using source at D:/soft/vcpkg/buildtrees/dataframe/src/head/9961acef64-2ad6077aff.clean
> -- Found external ninja('1.11.0').
> -- Configuring x86-windows
> CMake Warning at installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:314 (message):
>   The following variables are not used in CMakeLists.txt:
> 
>       ENABLE_TESTING
> 
>   Please recheck them and remove the unnecessary options from the
>   `vcpkg_cmake_configure` call.
> 
>   If these options should still be passed for whatever reason, please use the
>   `MAYBE_UNUSED_VARIABLES` argument.
> Call Stack (most recent call first):
>   ports/dataframe/portfile.cmake:8 (vcpkg_cmake_configure)
>   scripts/ports.cmake:147 (include)
> 
> 
> -- Building x86-windows-dbg
> -- Building x86-windows-rel
> CMake Error at installed/x64-windows/share/vcpkg-cmake-config/vcpkg_cmake_config_fixup.cmake:36 (message):
>   'D:/soft/vcpkg/packages/dataframe_x86-windows/debug/cmake' does not exist.
> Call Stack (most recent call first):
>   ports/dataframe/portfile.cmake:17 (vcpkg_cmake_config_fixup)
>   scripts/ports.cmake:147 (include)
> 
> 
> error: building dataframe:x86-windows failed with: BUILD_FAILED
> internal error: D:\a\_work\1\s\src\vcpkg\build.cpp(1475): Value was null
> Please open an issue at https://github.com/microsoft/vcpkg/issues/new?template=other-type-of-bug-report.md&labels=category:vcpkg-bug with detailed steps to reproduce the problem.
> PS D:\soft\vcpkg>

--- a/ports/dataframe/portfile.cmake
+++ b/ports/dataframe/portfile.cmake
@@ -1,25 +1,22 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO hosseinmoein/DataFrame
-    REF 1.19.0
-    SHA512 4415644b04d9c3b5ab3487ee896d0f020f0659d71b8bc128a649e14c6edb22c4848d228016a0108a122060d5aa9452260a3936761cf228eb7a7a436a64f031ff
+    REF "${VERSION}"
+    SHA512 e37df83396746620d4d1de3a9ef20cf4beea5c47970da2c9c368aa56a2a783396658a218d922a6e840fd1ad2a0aa54d4ad8dfbfe3b33b555efae6fe2deea00b5
     HEAD_REF master
 )
 vcpkg_cmake_configure(
     SOURCE_PATH "${SOURCE_PATH}"
     OPTIONS
-        -DENABLE_TESTING:BOOL=OFF
+        -DHMDF_TESTING:BOOL=OFF
 )
 
 vcpkg_cmake_install()
 
-if(VCPKG_TARGET_IS_WINDOWS)
-    vcpkg_cmake_config_fixup(CONFIG_PATH cmake)
-else()
-    vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/DataFrame)
-endif()
+vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/DataFrame)
+
 vcpkg_fixup_pkgconfig()
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/include")
 
-file(INSTALL "${SOURCE_PATH}/License" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/License")

--- a/ports/dataframe/vcpkg.json
+++ b/ports/dataframe/vcpkg.json
@@ -1,8 +1,9 @@
 {
   "name": "dataframe",
-  "version": "1.19.0",
+  "version": "1.22.0",
   "description": "This is a C++ statistical library that provides an interface similar to Pandas package in Python",
   "homepage": "https://github.com/hosseinmoein/DataFrame",
+  "license": "BSD-3-Clause",
   "supports": "!uwp",
   "dependencies": [
     {

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -1905,7 +1905,7 @@
       "port-version": 1
     },
     "dataframe": {
-      "baseline": "1.19.0",
+      "baseline": "1.22.0",
       "port-version": 0
     },
     "date": {

--- a/versions/d-/dataframe.json
+++ b/versions/d-/dataframe.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "1dbcb0119d811c5e0ef7e36d5ee456396ac3c10a",
+      "version": "1.22.0",
+      "port-version": 0
+    },
+    {
       "git-tree": "7f3b69090a3dce2e274e137efce5c2fb859d25ef",
       "version": "1.19.0",
       "port-version": 0

